src/css/betterttv.css

```diff
@@ -157,8 +153,16 @@ input.text:focus,select:focus,textarea:focus {
    text-decoration: none;
 }
 
-.ember-chat .badges, .ember-chat .mod-icons {
-    font-size: 0;
+.ember-chat .badges span:last-child .badge {
```

Comment:
All alignment issues stemmed from `font-size: 0` on both the mod icons and badges. With it being removed, the chat line goes back to its former alignment. All vertical margin tweaking is because of the compensation for my mistake. This change is mainly for the left and right additional whitespace added with `span` in an `inline-block` display. 

While I didn't want to resort to margins. `font-size: 0` was a mistake on multiple fronts.